Translation corrections (#192)
07th-mod/higurashi-console-arcs: Update/prol_002.txt

//r自分の担当事件で傷つき斃れるのは、この職業につきまとう因果だからそれも本望だろう。……だが、彼は違った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "自分の担当事件で傷つき斃れるのは、この職業につきまとう因果だからそれも本望だろう。……だが、彼は違った。",

> 			NULL, "Getting injured and dying on your case is an occupational hazard in this line of duty, and I probably would have welcomed it. ...But it was different in his case.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Getting injured and dying while working a case is an occupational hazard in this line of duty, and I probably would have welcomed it. ...But it was different in his case.", Line_Normal);
```

//r……顔も知らないやつらのあざ笑う声が聞こえてくるようで、思わず固めたこぶしに力が入った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "……顔も知らないやつらのあざ笑う声が聞こえてくるようで、思わず固めたこぶしに力が入った。",

> 			NULL, "...I could hear the jeering voices of those people whose faces I didn't knowe, and suddenly began to clench my fists.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...I could hear the jeering voices of those people whose faces I didn't know, and suddenly began to clench my fists.", Line_Normal);
```
